docs/reference/upgrade/reindex_upgrade.asciidoc

```diff
@@ -112,6 +112,14 @@ indices from the previous major version to be upgraded to the
 current major version. Skipping a major version means that you must
 resolve any backward compatibility issues yourself.
 
+ifdef::include-xpack[]
+If you use {ml-features} and you're migrating indices from a 6.5 or earlier
+cluster, the job and {dfeed} configuration information are not stored in an
+index. You must recreate your {ml} jobs in the new cluster. If you are migrating from a 6.6 or later cluster, it is a good idea to temporarily halt the tasks associated with your {ml} jobs and {dfeeds} and prevent new jobs from opening
```

Comment:
The jobs won't open on the new cluster because whether jobs are open or closed is determined by persistent tasks that won't be brought across by the reindex.

So I would change "prevent new jobs from opening" to something like "prevent inconsistencies between different ML indices that are reindexed at slightly different times".

Reply:
Thanks @droberts195 does that clarification apply to the "reindex in place" section too?

Reply:
For reindex in place the persistent tasks are in the same cluster, so I think that clarification only applies to reindex from remote.